services/workflows-service/src/alert/alert.controller.external.ts

```diff
@@ -201,4 +207,15 @@ export class AlertControllerExternal {
       }),
     };
   }
+
+  @common.Get('/:id/alert-definition')
```

Comment:
@alonp99 @liorzam let me know if the path is fine with you.

Reply:
LGTM